rust/sealevel/programs/mailbox/src/accounts.rs

```diff
@@ -0,0 +1,132 @@
+//! Hyperlane Sealevel Mailbox data account layouts.
+
+use std::{collections::HashSet, str::FromStr as _};
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use hyperlane_core::{accumulator::incremental::IncrementalMerkle as MerkleTree, H256};
+use solana_program::{
+    account_info::AccountInfo,
+    program_error::ProgramError,
+    // Note: Not convinced program_pack::{IsInitialized, Pack} add value here.
```

Comment:
This still relevant?

Reply:
I don't think those traits are particularly useful so if you don't have an opinion, I'll delete